tools/wake/main.cpp

```diff
@@ -235,6 +238,7 @@ int main(int argc, char **argv) {
   const char *fd3     = arg(options, "fd:3")->argument;
   const char *fd4     = arg(options, "fd:4")->argument;
   const char *fd5     = arg(options, "fd:5")->argument;
+  const char *build_vis_file = arg(options, "build-visualization")->argument;
```

Comment:
Make this a  'bool' (->count)

Reply:
Maybe something like `--gantt` or `--timeline` or something memorable and easy to type.

Reply:
Done